### PR TITLE
UCSC-98 Add Main nav block

### PR DESCRIPTION
**Pull request by MlKilderkin**
## What 
Register new nav block for primary navigation menu

## Ticket

https://moderntribe.atlassian.net/browse/BM2-937

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html <?php language_attributes(); ?>>
 <head>
 	<meta charset="<?php bloginfo( 'charset' ); ?>">
-
+<?php var_dump(__DIR__);?>
```
Comment (bjcooper) [inline index.php:5]: Should this still be here?

**Review by bjcooper — APPROVED** [review]
🟡 Very cool, Myke! There's just one `var_dump` to maybe check that it belongs, otherwise this looks good to me!

File: blocks/main-nav/index.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/**
+ * Scripts specific to this block
+ */
+
+import './style.pcss';
```
[the remark this answers is not in the document]
Comment (Vinsanity) [reply blocks/main-nav/index.js:5]: @bjcooper @MlKilderkin  `wp-scripts` is still not fully supporting pcss which is why we're using scss for all blocks. I do not recommend making that change on this project.

[the remark this answers is not in the document]
Comment (bjcooper) [reply blocks/main-nav/index.js:5]: OK! For clarity, I am planning to replace this PostCSS with Sass equivalent when I get it into FE dev.

**Pull request merged**